### ISSUE TITLE
代码字体大小修改后火狐浏览器渲染失败

### ISSUE THREAD

**Issue by swordspoet** [iissnan/hexo-theme-next#1647]
### Expected behavior (预期行为)

代码字体大小修改为13.5px

### Actual behavior (实际行为)

![Firefox](https://cloud.githubusercontent.com/assets/14816130/25652651/d32414c4-301b-11e7-8616-2f1cc792aa9c.png)

![chrome](https://cloud.githubusercontent.com/assets/14816130/25652633/b10d6dc2-301b-11e7-96a7-3118fc99c03b.png)

### NexT Information

Add the  ✔  sign before an item which is affected by this behavior.

NexT Version:

  -  Master✔ 
  -  Latest Release
  -  Old version - 

NexT Scheme:
  -  All schemes
  -  Muse
  -  Mist✔ 
  -  Pisces

**Comment by ivan-nginx** [comment]
Can u explain your issue in English?

**Comment by swordspoet** [comment]
@ivan-nginx ok, the  default code font size is 13px, but it seems too small to read it. I changed the code font size to 13.5px, after 'hexo g ', it was going well both on chrome and firework, it's strange that problem thrown out as the screen shoot when I 'hexo g  -d'. I have no idea what the problem was.

Hope you can get what I am saying

**Comment by ivan-nginx** [comment]
@swordspoet before start resolve this, u try to clean and after that regenerate?
```
hexo clean && hexo g && hexo d
```

**Comment by swordspoet** [comment]
@ivan-nginx yes, I am sure I have done that, I guess maybe it was firefox's problem

**Comment by ivan-nginx** [comment]
@swordspoet ok, can u see some scrinshots or anything to see and compare firefox and chrome fonts?

**Comment by ivan-nginx** [comment]
@swordspoet solved?
